chain/stmgr/stmgr.go

```diff
@@ -189,7 +189,7 @@ func (sm *StateManager) ApplyBlocks(ctx context.Context, pstate cid.Cid, bms []B
 			Miner:       b.Miner,
 			Penalty:     penalty,
 			GasReward:   gasReward,
-			TicketCount: b.TicketCount,
+			TicketCount: 1, // TODO: no longer need ticket count here.
```

Comment:
Remove

Reply:
After spec-actors update.

Reply:
Does doing this before updating break anything?

Reply:
nope, it just gives out a single block reward to all valid block miners.